extension/redirect-to-arcs.html

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright (c) 2017 Google Inc. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt
+Code distributed by Google as part of this project is also
+subject to an additional IP rights grant found at
+http://polymer.github.io/PATENTS.txt
+-->
+<!doctype html>
+<html lang="en">
```

Comment:
I think the html and body tags are optional if you've got the doctype tag, see:

https://stackoverflow.com/a/25749523

Reply:
!

Reply:
It looks kind of bare, but it's much shorter.